services/data-ingestor/main.py

```
@@ -28,45 +35,118 @@ def health_check():
     load_dotenv(dotenv_path=dotenv_path)
 
 # ----------------------------------------
-# Add shared libraries to sys.path
+# Import diagnostics module or fallback
 # ----------------------------------------
-libs_path = os.path.abspath(os.path.join(os.path.dirname(__file__), "../../libs"))
```

Just flagging that this is useful when running main.py manually outside Docker (e.g., during testing or debugging). I suggest keeping it for dev convenience, otherwise, local runs might silently fail due to missing env vars.